com.unity.netcode.gameobjects/Components/NetworkTransform.cs

```diff
@@ -390,6 +398,16 @@ private void ResetInterpolatedStateToCurrentAuthoritativeState()
             m_ScaleZInterpolator.ResetTo(m_LocalAuthoritativeNetworkState.ScaleZ, serverTime);
         }
 
+        /// <summary>
+        /// Will apply the transform to the LocalAuthoritativeNetworkState and get detailed isDirty information returned.
+        /// </summary>
+        /// <param name="transform">transform to apply</param>
+        /// <returns>bool isDirty, bool isPositionDirty, bool isRotationDirty, bool isScaleDirty</returns>
+        internal (bool isDirty, bool isPositionDirty, bool isRotationDirty, bool isScaleDirty) ApplyLocalNetworkState(Transform transform)
```

Comment:
I know this is following the paradigm but at this point, should this mega-tuple be turned into a struct? The tuple signature is replicated at least 3 times in this file, and with this internal API you're exposing the tuple to outside classes.

Reply:
Yeah... I know... :(
I am trying to be very selective about what I am changing in these PRs...actually this PR is only a portion of #1846 just so I can get this rotation fix in without any other changes blocking this (a higher priority fix).

@ThusWroteNomad What do you think?  Switch that to a struct in this PR or make that a separate PR?

Reply:
I think a nice and simple separate PR proposing tuple -> struct would be great :)

Reply:
Ok, once this gets merged I will whip up a quick PR for that.